### ISSUE TITLE
oh-my-zsh compatibility

### ISSUE THREAD

**Issue by robbielove** [nrobinson2000/po-util#45]
Could you please make this compatible with oh-my-zsh, I had difficulty understanding why this wouldnt work after being installed but discovered during the install that it couldnt find my .bashrc file 
```
...
Shoud po-util automatically add and remove headers when using
    libraries?
(yes/no): yes

grep: /Users/username/.bashrc: No such file or directory
...
```

prompting me to use bash which worked fine
```
bash-3.2$ po init photon newProject

Directory initialized as a po-util project for photon
```

however when previously using oh-my-zsh, I was getting the following error:

```
po install
~/dev ~

po init photon newProject       
popd: too many arguments
```

**Comment by nrobinson2000** [comment]
Thanks for posting your errors.

```
grep: /Users/username/.bashrc: No such file or directory
```

This can be safely ignored. Po is trying to enable tab completion by adding an entry to your `.bashrc` file. If `~/.profile` or `.bash_profile` exist, (which apparently do), then tab completion should be enabled.

```
po init photon newProject
```

This work fine, but you can also use `po photon init newProject`. I need to work on the documentation some more.

```
po init photon newProject       
popd: too many arguments
```

For some odd reason oh-my-zsh believes that it is a reasonable idea to make `po` be an alias for `popd`, a command that is hardly ever used.

I've addressed this issue here:
https://github.com/nrobinson2000/homebrew-po/issues/7

Basically, you could put `unalias po` in one of your `.profile` or preferred file to disable the alias, or use the `po-util` alias, which aliases to `/usr/local/bin/po` which is what `po` is supposed to do.

**Comment by robbielove** [comment]
Thanks, placing `unalias po` at the end of my `~/.zshrc` file worked

**Comment by nrobinson2000** [comment]
I reached out to oh-my-zsh maintainers and made a pull request to remove the `po` alias. This problem should be resolved.
